lesson_09-HW_19/index.js

```diff
@@ -0,0 +1,74 @@
+class Person {
+   name;
+   age;
+
+   constructor(name, age) {
+      this.name = name;
+      this.age = age;
+   }
+}
+
+class Flat {
+   residents = [];
+
+   addResidents(person) {
+      this.residents.push(person)
+   }
+
+   logAllPerson() {
```

Comment:
```suggestion
   logAllPersons() {
```
а тут якраз виводити багатьох)

Reply:
Вивчення англійскоЇ в процесі)

виправив)